src/coinjoin/client.cpp

```diff
@@ -1363,9 +1363,9 @@ bool CCoinJoinClientSession::PrepareDenominate(int nMinRounds, int nMaxRounds, s
         return true;
     }
 
-    for (const auto& pair : vecPSInOutPairsRet) {
-        mixingWallet.LockCoin(pair.first.prevout);
-        vecOutPointLocked.push_back(pair.first.prevout);
+    for (const auto& inout : vecPSInOutPairsRet) {
+        mixingWallet.LockCoin(inout.first.prevout);
+        vecOutPointLocked.push_back(inout.first.prevout);
```

Comment:
```suggestion
    for (const auto& [txDsIn, _] : vecPSInOutPairsRet) {
        mixingWallet.LockCoin(txDsIn.prevout);
        vecOutPointLocked.push_back(txDsIn.prevout);
```

Reply:
`_` is not used variable, and [[unused]] is not allowed to be used inside structure binding.
I want to keep this case as it is.

Reply:
We do this in quite a few places already. Why would this be an issue?

Reply:
Agree with Udjin here

Reply:
```
strAutoDenomResult = _("Session timed out.");                                  
```
`_` is used in this file and variable with name "_" will shadow it.

Reply:
Hmm.. That's a valid point I guess 👍